vsintegration/src/FSharp.VS.FSI/Properties.resx

```diff
@@ -118,10 +118,10 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="FSharpInteractiveAnyCpu" xml:space="preserve">
-    <value>Use fsiAnyCpu.exe</value>
+    <value>Run as 64-bit</value>
   </data>
   <data name="FSharpInteractiveAnyCpuDescr" xml:space="preserve">
-    <value>If set to true, 'fsiAnyCpu.exe' is run as a platform neutral process. Otherwise, F# Interactive is a 32-bit process on X86/X64 or a native 64 bit process on Arm64 based systems.</value>
+    <value>If set to true, F# Interactive is a 64-bit process. Otherwise, F# Interactive is 32-bit on X86/X64 and 64-bit on Arm64.</value>
```

Comment:
So a developer on an arm64 would read this as:
**If set to true, F# Interactive is a 64-bit process. Otherwise, F# Interactive is** 32-bit on X86/X64 and **64-bit on Arm64.**
I.e. on an Arm64
True = 64-bit, False = 64 Bit

In fact the difference is a bit more striking than that.  **True** = Emulated X64 on ARM 64, **False** = Native Arm64-bit.
If we are really going to update the value.

And all of this is moot if we are using the dotnet sdk for fsi.  Where this option is ignored.

I'm not against changing the text per se, but whichever we go is going to cause confusion.

The rules are:
1. If use dotnetsdk this option is ignored.
2. On x86/64 set to **True** --- Uses FsiAnyCpu.exe which executes natively on the 64 bit CLR
3. On x86/64 set to **False** --- Uses fsi.exe which executes natively on the 32 bit CLR
4. On Arm64 set to **True** --- Uses FsiAnyCpu.exe which executes emulated on the 64 bit CLR
5. On Arm64 set to **False** --- Uses fsiArm64.exe which executes natively on the 64 bit Arm CLR

Therefor the choice of the developer should be along the lines of:

What dependencies is my script using:
1. Do I have native 64 bit X64 desktop CLR libraries to run?  If yes then set flag to **true** (*Regardless of Processor type *)
2. Do I have native 64 bit Arm64 desktop CLR libraries to run?  If yes then set flag to **false** (* Also requires an Arm64 machine *)
3.Do I have native X86 desktop CLR libraries to run?  If yes then set flag to **false** (* Also requires an X86/64 machine *)
4. In all other circumstances use the dotnet SDK.

Reply:
Oh, wow.

What if we changed this into a dropdown listing the 4 options you mentioned, rephrased as "Are you using any native dependencies?", with default of "No".
And the options would be the special kinds of dependencies, resolved by us into the boolean flags.

Reply:
Personally I want to get rid of the desktop build, and have done so for a long time.